install/install-kali-2016-2.sh

```diff
@@ -4,6 +4,9 @@ sudo apt-get install jq nmap libgeoip-dev libffi-dev libcurl4-openssl-dev libxml
 
 pip install requests
 
+cd ..
```

Comment:
Nope! Look at the lines below, this script assumes that you're in the root directory of the project so you don't want the `../` there.

That being said, amending the script to adjust to a proper directory to avoid errors based on where you're running it from would be nice... or give it a "target" argument or something.